src/navigator/Navigator.tsx

```diff
@@ -289,11 +291,13 @@ const sendScreens = (Navigator: typeof Stack) => (
     />
     <Navigator.Screen
       name={Screens.IncomingPaymentRequestListScreen}
+      // @ts-ignore
```

Comment:
nit: once this become unnecessary TS will remind us to remove it
```suggestion
      // @ts-expect-error
```

Reply:
Fixed in 2f323b52ff25806a3cc1185b0b2c3182943ba035!